components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -3051,8 +3051,6 @@ public GroupsGetResponse listGroupsWithGET(Node rootNode, Integer startIndex, In
         startIndex = handleStartIndexEqualsNULL(startIndex);
         if (sortBy != null || sortOrder != null) {
             throw new NotImplementedException("Sorting is not supported");
-        } else if (startIndex != 1 && count != null) {
```

Comment:
Let's remove this in the above deprecated method as well

Reply:
Fixed by - https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/620/commits/7385e8b1c4ec54afa669f162e2e57e54a5776b25